test/annotation/image_classifier_tests.cxx

```diff
@@ -346,7 +346,7 @@ struct image_classification_test {
 
     TS_ASSERT(annotation_testing::check_equality(annotation_sf, returned_sf));
 
-    
+
```

Comment:
Since this is only a whitespace change, please revert the changes in this file.

Reply:
Yeah, I could. IMO, it can keep the source pkg smaller XD. Besides, you can set ignore whitespaces change when you do code review. 
https://github.blog/2018-05-01-ignore-white-space-in-code-review/
@znation